kafka-connect-s3/src/main/java/io/confluent/connect/s3/TopicPartitionWriter.java

```diff
@@ -656,8 +656,8 @@ protected synchronized void commitFiles() {
 
     // Make sure to commit the offset to kafka in case kafka fails after the commit to S3
     if (recordCount == 0) {
-      log.info("No File to commit to S3. Target commit offset for {} is {}", tp, offsetToCommit);
       offsetToCommit = currentOffset + 1;
+      log.info("No files to commit to S3. Target commit offset for {} is {}", tp, offsetToCommit);
```

Comment:
Fix some mistake from my previous commit (I logged before I set the value so I always got `null` for `offsetToCommit`)